packages/drivers/routerlicious-driver/src/urlUtils.ts

```diff
@@ -17,3 +19,25 @@ export const parseFluidUrl = (fluidUrl: string): URLParse => {
  */
 export const replaceDocumentIdInPath = (urlPath: string, documentId: string): string =>
     urlPath.split("/").slice(0, -1).concat([documentId]).join("/");
+
+export const createFluidUrl = (domain: string, pathname: string): string =>
+    "fluid://".concat(domain, pathname);
+
+export const replaceWithDiscoveredUrl = (resolvedUrl: IFluidResolvedUrl,
+    session: ISession,
+    parsedUrl: URLParse): void => {
+    if (session && session.ordererUrl.includes("https")) {
```

Comment:
Why do we check that ordererUrl is HTTPS? This would prevent local testing of the discovery endpoint to fail. Also, if the service is returning an HTTP endpoint, that is the service's decision and likely should not be verified here without good reason.